src/prefect/utilities/notifications/notifications.py

```diff
@@ -125,7 +125,12 @@ def slack_message_formatter(
     tracked_obj: TrackedObjectType,
     state: "prefect.engine.state.State",
     backend_info: bool = True,
+    template: str = None,
+    template_vars: Mapping = None,
 ) -> dict:
+    template = template or f"{tracked_obj} is now in a {state} state"
```

Comment:
Where should we define what names will be provided by default? What if a user wants to add to the template without having to recreate this string?

Reply:
Note to include prefect context variables

Reply:
I think that mentioning it in the docstring will suffice. The only things we have available to pass into the format call are those two vars and context